src/containers/TagManager.jsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Radium from 'radium';
+
+import Page from './Page';
+import ContentHeader from '../components/ContentHeader';
+import InPlaceEditor from '../components/forms/InPlaceEditor';
+import Tagger from '../components/forms/Tagger';
+
+import settings from '../settings';
+
+require('../../css/react-tag-input.css');
+
+@connect(state => {
+  return state.pipelines;
+})
+@Radium
+export default class TagManager extends React.Component {
+
+  componentWillMount() {
+    this.setState({ tags: [ 'top_commenter', 'banned', 'moderator', 'martian', 'ufo' ] });
+  }
+
+  confirmDeletion(index) {
+    var tagsCopy = this.state.tags;
+    tagsCopy.splice(index, 1);
+    this.setState({ tags: tagsCopy });
+  }
+
+  tagAdderChangeHandler(event) {
+    this.setState({ adderValue: event.target.value });
+  }
+
+  tagAdderClickHandler() {
+    if (this.state.adderValue) {
+      this.setState({ tags: [ ...this.state.tags, this.state.adderValue ] });
+      React.findDOMNode(this.refs.tagAdderInput).value = ''; 
+    }
+  }
+
+  tagAdderKeyHandler(event) {
+    if (event.key == 'Enter') {
+      this.tagAdderClickHandler();
+    }
+  }
+
+  onEditorSave(index, value) {
+    var firstSlice = this.state.tags.slice(0, index);
+    var lastSlice = this.state.tags.slice(index + 1);
+    var tagsCopy = firstSlice.concat(value).concat(lastSlice);
+    this.setState({ tags: tagsCopy });
+  }
+
+  render() {
+
+    return (
+
+      <Page>
+
+        <ContentHeader title="Tag Manager" />
+
+        <div style={ styles.tagAdder }>
+          <input style={ styles.tagAdderInput } ref="tagAdderInput" onChange={ this.tagAdderChangeHandler.bind(this) } onKeyPress={ this.tagAdderKeyHandler.bind(this) } type="text" placeholder="Add a new tag..." />
+          <button style={ styles.tagAdderButton } onClick={ this.tagAdderClickHandler.bind(this) }>Add tag</button>
+        </div>
+
+        <table style={ styles.tableBase }>
```

Comment:
instead of using a table here, check out [Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) as a layout mechanism. It's way simpler semantically, and Radium takes care of all the vendor prefixing for you!